spec/index.bs

```diff
@@ -15895,9 +15893,16 @@ It guarantees that:
 - if |alpha| is greater than some other |alpha1|,
     then the produced sample mask has at least as many bits set to 1 as the mask for |alpha1|
 
-### Sample frequency shading ### {#sample-frequency-shading}
+### Per-Sample shading ### {#per-sample-shading}
+<span id="sample-frequency-shading"></span>
```

Comment:
nit: This span should be inside the header otherwise the link will take you just below the header, instead of to the header. (see other instances for examples)

(replace the `###` with an `<h4>` which is what `###` means in bikeshed)